domain/model/item.go

```diff
@@ -0,0 +1,7 @@
+package model
+
+// Se deja el nombre generico item en lo que se encuentra un correcto nombre para el modelo de negocio
```

Comment:
Remember the format on how to document your exported items; let's try to follow these best practices.

Also, IMHO, I don't like the idea of mixing languages (English and Spanish); even these are comments, people who don't know Spanish will be struggling to read them.